pants.toml

```diff
@@ -66,6 +66,24 @@ build_ignore.add = [
   "testprojects/src/go/**",
 ]
 
+#            ___________    ____
+#     ______/   \__//   \__/____\
+#   _/   \_/  :           //____\\
+#  /|      :  :  ..      /        \
+# | |     ::     ::      \        /
+# | |     :|     ||     \ \______/
+# | |     ||     ||      |\  /  |
+#  \|     ||     ||      |   / | \
+#   |     ||     ||      |  / /_\ \
+#   | ___ || ___ ||      | /  /    \
+#    \_-_/  \_-_/ | ____ |/__/      \
+#                 _\_--_/    \      /
+#                /____             /
+#               /     \           /
+#               \______\_________/
+#
```

Comment:
huh?

Reply:
> (temporarily touching pants.toml to force CI to build+test wheels)

Reply:
> huh? nevermind

was my reaction too... but I was "fortunate" enough to spot Josh's comment before saying anything ;)